### PR TITLE
covergae for python integration tests

### PR DESCRIPTION
**Pull request by shay6**

### CODE REVIEW

File: documentation/python/README.md [hunk 1]
```diff
@@ -715,3 +715,131 @@ The following file illustrates a possible JJB configuration
             sonar.exclusions=tests/**/*.py
             sonar.ws.timeout=180
 ```
+#Code Coverage With Integration Tests
+
+Reference Docs:
+
+ - [How to generate coverage report for http based integration tests?](https://stackoverflow.com/questions/50689940/how-to-generate-coverage-report-for-http-based-integration-tests)
```
Comment (abraverm) [inline documentation/python/README.md]: For keeping 80 chars limit you can put the reference at the buttom:
```
[How to generate coverage report for http based integration tests?]
...
[How to generate coverage report for http based integration tests?]: https://stackoverflow.com/questions/50689940/how-to-generate-coverage-report-for-http-based-integration-tests
```

File: documentation/python/README.md [hunk 2]
```diff
@@ -715,3 +715,131 @@ The following file illustrates a possible JJB configuration
...
+
+ - [pytest](https://docs.pytest.org/en/latest/)
+ - [pytest-cov](https://pypi.org/project/pytest-cov/)
+
+We can implement the solution using the next steps:
+
+1. Run the Server under Coverage mode.
+2. Run the tests.
+3. Ensure the Server coverage is written to file.
+4. Read the coverage from this file and append it to the tests coverage report.
```
Comment (abraverm) [inline documentation/python/README.md]: breake line

File: documentation/python/README.md [hunk 3]
```diff
@@ -715,3 +715,131 @@ The following file illustrates a possible JJB configuration
...
+
+
+def test_get():
+    respond = requests.get('http://127.0.0.1:7000')
+    respond.raise_for_status()
+    assert 'Hello World' in respond.text
+```
+###Solution
+
+Below it is a [conftest.py](https://docs.pytest.org/en/2.7.3/plugins.html?highlight=re#conftest-py-plugins) file located under "tests" folder and
```
Comment (abraverm) [inline documentation/python/README.md]: break line and ref at the bottom

**Review by abraverm — CHANGES_REQUESTED** [review]
the content looks good

File: documentation/python/README.md (hunk 3 above)
Comment (shay6) [reply documentation/python/README.md]: done

File: documentation/python/README.md (hunk 2 above)
Comment (shay6) [reply documentation/python/README.md]: done

File: documentation/python/README.md (hunk 1 above)
Comment (shay6) [reply documentation/python/README.md]: done

**Review by abraverm — APPROVED** [review]

**Pull request merged**